deptrac.config.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+use Internal\Qossmic\Deptrac\IgnoreDependenciesOnContract;
+use Internal\Qossmic\Deptrac\IgnoreDependenciesOnShouldNotHappenException;
+use Qossmic\Deptrac\Contract\Analyser\ProcessEvent;
+use Qossmic\Deptrac\Core\Layer\Collector\CollectorType;
```

Comment:
Classes in `Qossmic\Deptrac\Core` or `Qossmic\Deptrac\Supportive` need to be moved into `Qossmic\Deptrac\Contract` to be used here.

Reply:
I will if the decide on this current implementation of the `DeptracConfig`-Class.